rpm/BUILD.bazel

```diff
@@ -272,6 +272,8 @@ rpmtree(
         "/var/run": "../run",
         "/usr/sbin/iptables": "/usr/sbin/iptables-legacy",
         "/usr/bin/nc": "/usr/bin/ncat",
+        "/usr/bin/python": "/usr/bin/python3.6",
```

Comment:
Is this actually necessary? Having the "python" name refer to Python 3 rather than Python 2 is generally frowned upon, and also not what you usually encounter on either CentOS or Fedora.

Reply:
I only have python3 installed and there happen some `python` calls from bazel. I can check if I can improve the bazel python detection a little.

Reply:
Never mind, it looks like Fedora actually has `/usr/bin/python` pointing to Python 3, and in fact I had to create the same symlink for the CentOS Stream-based builder image because Bazel (unfortunately) requires that to work :(